src/elm/Session/LoggedIn.elm

```diff
@@ -478,6 +518,15 @@ viewFooter _ =
         ]
 
 
+getBackgroundColor : Bool -> String
+getBackgroundColor isSuccess =
+    if isSuccess == True then
+        "#8ACC9E"
+
+    else
+        "#DB1B1B"
```

Comment:
we should use tailwind classes for that `green` and `red`. Actually maybe we don't need a function to encapsulate this `if`

Reply:
I tried those classes but for some reason they wouldn't work. Maybe we're using an outdated version?